docs/source/introduction.rst

```diff
@@ -44,6 +44,11 @@ Running this example from your terminal should simply show you a log message
 saying `Hello world!` but, what exactly is happening there?
 
 
+.. note:: The ``ns.shutdown()`` call is present because we don't want examples
+   to run indefinetely. For more information on that method, refer to the
+   :ref:`shutdown_nameserver` section.
```

Comment:
I would rather point to the current "Shuttting down" section in the "Considerations" (as shutting-down is not specific to distributed systems).